apps/web/src/components/utils/GotAQuestionButton.tsx

```diff
@@ -0,0 +1,13 @@
+import { Button, Size } from '../../design-system';
+
+function onClick() {
+  window?.open('https://discord.novu.co', '_blank')?.focus();
```

Comment:
If somehow we would be able to add the channel to open automatically could be a cool improvement.